### DISCUSSION TITLE
What does "Organization Registration allowed" do?

### DISCUSSION THREAD

**Discussion by Tomomi-K1** [zitadel/zitadel#8395] [Q&A]
![image](https://github.com/user-attachments/assets/6b61ff7c-c322-41a8-b88c-101aa947abbd)

Under the default setting, Login Behavior and Secuerity, I see "Organization Registration allowed". When do we need to check this? Or when should not check this on. Is there any documentation about this ( if there is sorry, I couldn't find...)?

**Comment by hifabienne** [comment]
Hei @Tomomi-K1 
This does enable the path in the ui /ui/login/regier/org
Which allows your to let customers register a new organization with a user:

<img width="790" alt="image" src="https://github.com/user-attachments/assets/9595485d-1161-47aa-9024-6b5391e3fb02">

**Reply by alexcatdad** [reply]
Scenario: A user created an account via user registration without an organization
Question: How does that user now create an org?

**Reply by hifabienne** [reply]
That doesn't work. A user always belongs to an organization.
If a user clicks the registration button on the login page, it will be registered to the default organization.
If you pre select a login of another orgnaization, then the user will be registered to that.
Or you can use domain discovery to register a user to a specific organization. 
https://zitadel.com/docs/guides/solution-scenarios/domain-discovery

**Reply by hifabienne** [reply]
I you do want to have another registration page, you can also build it by yourself and just call the ZITADEL APIs

**Comment by Tomomi-K1** [comment]
@hifabienne Thank you for answering the question!!

I have followup questions:
1. Is there a environment variable to set this to false?

2. I'm trying to setup microsoft SSO for one organization. Single-tenant.
When do we choose `Tenant ID` as Tenant Type?
![image](https://github.com/user-attachments/assets/a7ec005e-0650-4581-8c31-ca2cbcdeba54)

**Reply by hifabienne** [reply]
1. I think the following ENV you can set it: ZITADEL_DEFAULTINSTANCE_RESTRICTIONS_DISALLOWPUBLICORGREGISTRATION
2. I think it is tenant id when you only want to allow users of a specific organization to be able to login
